app/Providers/AppServiceProvider.php

```diff
@@ -26,6 +27,9 @@ public function register(): void {
      * @throws BindingResolutionException
      */
     public function boot(): void {
+        if (config('app.env') === 'prod') {
```

Comment:
```suggestion
        if (config('app.env') === 'production') {
```
Sorry, I missed that last time. xD